### DISCUSSION TITLE
New Version of `config` Adapting BeitV2 Algorithm: 新版 config 适配 BeiTv2

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#335] [MMPreTrain]
# New Version of `config` Adapting BeitV2 Algorithm: 
 新版 config 适配 BeiTv2 

New Version of `config` Adapting BeitV2 Algorithm:
 MMEngine has introduced a novel configuration mechanism, utilizing which we will perform configuration file adaptation. 
 For reference, visit [folder]([https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)](https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)) 
 Target folders for this task's adaptation are 'configs/beit/' and 'configs/beitv2/' where all files need to be adapted.

新版 config 适配 beitv2 算法 
 mmengine 实现了一套新的 config 机制，基于新的 config 机制进行配置文件的适配，[相关案例]([https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)](https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)) 
 该任务适配的 config 目录：configs/beit/ 和 configs/beitv2/ ，完成所有文件适配

## Task Information

**Repository**：MMPreTrain

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：DeliMm

**Claim Time**：2023年08月01日

**PR Link**：https://github.com/open-mmlab/mmpretrain/pull/1755

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by DeliMm** [comment]
The PR link for this task is https://github.com/open-mmlab/mmpretrain/pull/1755. Thanks for giving me the opportunity to complete this task.

**Reply by crazysteeaam** [reply]
good job！
